Mathlib/Algebra/Module/Submodule/Invariant.lean

```diff
@@ -38,6 +38,18 @@ lemma mem_invtSubmodule {p : Submodule R M} :
 
 namespace invtSubmodule
 
+variable {f}
+
+lemma inf_mem {p q : Submodule R M} (hp : p ∈ f.invtSubmodule) (hq : q ∈ f.invtSubmodule) :
+    p ⊓ q ∈ f.invtSubmodule :=
+  ((⟨p, hp⟩ : f.invtSubmodule) ⊓ (⟨q, hq⟩ : f.invtSubmodule)).property
```

Comment:
I was wondering why we needed this, and then I found out that we have no `SublatticeClass` :scream: Obviously it's not something that should be handled in this PR.

Reply:
Yep. Actually sublattices arrived amazingly late to Mathlib and there is still a bunch of stuff missing (including structures for sublattices containing top and / or bottom).